src/openrct2/actions/LandSetHeightAction.cpp

```diff
@@ -246,7 +246,7 @@ money32 LandSetHeightAction::GetSmallSceneryRemovalCost() const
         if (sceneryEntry == nullptr)
             continue;
 
-        cost += (sceneryEntry->removal_price) * 10;
+        cost += toGBP(static_cast<long double>(sceneryEntry->removal_price));
```

Comment:
Not sure how this is easier to read than simply having `(sceneryEntry->removal_price) * 10` here like before. @Gymnasiast, were you thinking of another solution?

Reply:
I'm confused why this is necessary at all. Why doesn't `sceneryEntry->removal_price` already hold the money value is represents?

Reply:
> Not sure how this is easier to read than simply having `(sceneryEntry->removal_price) * 10` here like before. @Gymnasiast, were you thinking of another solution?

To be fair, I'm not 100% sure the `static_cast` is needed, it is converting from signed int to double, so it should be alright to have it without cast, which would make it a lot easier on the eyes.

> I'm confused why this is necessary at all. Why doesn't sceneryEntry->removal_price already hold the money value is represents?

I don't know myself, it's a fair question, the code was already doing this `* 10` multiplication, the patch just kept it 🤔

Reply:
I now understand it, I think. The objects store only the whole part of it, so if it costs 30 to remove a given object, the objects will store it as:
```
        "removalPrice": -52,
```

I guess it's just easier on maintaining the objects to have the wholes as they really are than to have them all multiplied by `10` and then explaining people that the rightmost digit is the decimal.

Reply:
we could *10 on load then change the type to money32 on the object. That would simplify all of this.

Reply:
That seems like a much better solution.

Reply:
I agree @duncanspumpkin , but it seems like too much for this PR, specially as it will break replays, won't it? There are at least four fields to change, `prince` and `removal_price` of small and large scenery.

I can happily tackle that in a different, follow-up PR.

Reply:
Of course a follow-up PR is better, this PR is just about getting the user defined literals in. `toGBP` is a nice substitute for now, and makes it easy to find which places need refactoring.

One nitpicky remark: functions should use TitleCase, `ToGBP`.

Reply:
> toGBP is a nice substitute for now, and makes it easy to find which places need refactoring.

Yep, I even searched the code for `* 10` and found more places that can be potentially simplified later on.

> One nitpicky remark: functions should use TitleCase, `ToGBP`.

Definitely, nice catch.